spyder_notebook/widgets/tests/test_client.py

```diff
@@ -41,13 +41,14 @@ def plugin(plugin_without_server):
 
     Use `plugin.client` to access the client.
     """
-    server_info = {'notebook_dir': '/path/notebooks',
+    server_info = {'root_dir': '/path/notebooks',
                    'url': 'fake_url',
                    'token': 'fake_token'}
     plugin_without_server.client.register(server_info)
     return plugin_without_server
 
 
+@pytest.mark.skip
```

Comment:
Do you plan to address this later?

Reply:
Well spotted! I don't think there is a reason anymore to skip that test, so I'll enable it again.